app/templates/courses/courses-view.jade

```diff
@@ -45,42 +45,10 @@ block content
 
     else
 
-      - var showHOCComplete = false;
-      if view.hocCourseInstance
-        - var course = view.courses.get(view.hocCourseInstance.get('courseID'));
-        - var campaign = view.campaigns.get(course.get('campaignID'));
-        - var stats = campaign.statsForSessions(view.hocCourseInstance.sessions);
-        - showHOCComplete = stats.levels.done && !view.classrooms.size();
-
       .text-center
-        if !showHOCComplete
-          h1(data-i18n="courses.welcome_to_page") Welcome to your Courses page!
-        else
-          h1(data-i18n="courses.completed_hoc")
-          h2(data-i18n="courses.ready_for_more_header")
-          ul.text-left
-            li(data-i18n="courses.ready_for_more_1")
-            li(data-i18n="courses.ready_for_more_2")
-            li(data-i18n="courses.ready_for_more_3")
-          a.btn.btn-lg.btn-success(href="/play") Play Now
-
-      if view.hocCourseInstance && !view.classrooms.size()
-        h3(data-i18n="courses.saved_games")
-        hr
+        h1(data-i18n="courses.welcome_to_page") Welcome to your Courses page!
 
-        .course-instance-entry
-          h3
-            span(data-i18n="courses.hoc")
-            span.spr :
-            span.spr(data-i18n="courses.course")
-            span 1
-            span.spr= (me.get('aceConfig') || {}).language === 'javascript' ? 'JavaScript' : 'Python'
-            small
-              a#change-language-link(data-i18n="courses.change_language")
-          +course-instance-body(view.hocCourseInstance)
-          .clearfix
-
-      else if view.classrooms.size()
+      if view.classrooms.size()
```

Comment:
I'd add `> 0` since negative numbers are truthy and you probably don't want those here either.